examples/generic_array_access.no

```diff
@@ -1,5 +1,5 @@
-fn last<NN>(arr: [Field; NN]) -> Field {
-    return arr[NN - 1];
+fn last(arr: [Field; N]) -> Field {
+    return arr[N - 1];
```

Comment:
since we're yelling on one-letter variables, I think we should do the same on one-letter numeric generics, wdyt?

Reply:
Agreed. 
Actually it is already updated to allow one-letter numeric generic: https://github.com/zksecurity/noname/pull/153/files#diff-52652f71bed52ba289f8c2066c3f850de28f9c099e199f94f305b097212a915a

Reply:
no I meant we should forbid one letter numerics, to encourage real variable names. Like `Len` instead.

Reply:
but I don't know if it's a good idea :D I don't know... wdyt?